internal/common/retrystrategy/retry_state.go

```diff
@@ -1,11 +1,18 @@
 package retrystrategy
 
 const (
-	RetryStrategyPendingState   = "PENDING"
-	RetryStrategyCompletedState = "COMPLETED"
-	RetryStrategyErrorState     = "ERROR"
-	RetryStrategyPausedState    = "PAUSED"
-	RetryStrategyUpdatingState  = "UPDATING"
-	RetryStrategyIdleState      = "IDLE"
-	RetryStrategyDeletedState   = "DELETED"
+	RetryStrategyPendingState    = "PENDING"
+	RetryStrategyCompletedState  = "COMPLETED"
```

Comment:
these consts are in `retrystrategy`package so starting with `RetryStrategy... ` seems redundant

Reply:
will address this in follow-up PRs as would not like to make this PR bigger for now

Reply:
sure, thanks